src/vc.rs

```diff
@@ -460,13 +460,28 @@ impl TryFrom<String> for URI {
     }
 }
 
+impl FromStr for URI {
```

Comment:
I thought these traits were already implemented?

Reply:
`TryFrom<String>` was implemented, but `FromStr` is separate. In this case `FromStr` is implemented using `TryFrom<String>`.

Reply:
Right my bad